npm.README.md

```diff
@@ -1,9 +0,0 @@
-### Installation
-
-```
-npm install --save @lifi/types
-```
-
-### Summary
```

Comment:
Why is this file removed?

Reply:
To simplify the release config, we don't have different READMEs for other repos and maintaining it is just an overhead.